src/main/java/uk/gov/digital/ho/hocs/info/client/auditClient/AuditClient.java

```diff
@@ -143,6 +145,136 @@ public void deleteTeamPermissionsAudit(UUID teamUUID, Set<PermissionDto> permiss
         }
     }
 
+    public void removeUserFromTeamAudit(UUID userUUID, UUID teamUUID) {
+        String auditPayload = Json.createObjectBuilder().add("teamUUID", teamUUID.toString()).add("userUUID", userUUID.toString()).build().toString();
+        CreateAuditRequest request = generateAuditRequest(auditPayload, EventType.REMOVE_USER_FROM_TEAM.toString());
+
+        try {
+            producerTemplate.sendBody(auditQueue, objectMapper.writeValueAsString(request));
+            log.info("Create audit for Remove User from Team, user UUID: {}, team UUID: {}, correlationID: {}, UserID: {}",
+                    userUUID, teamUUID, requestData.correlationId(), requestData.userId(), value(EVENT, AUDIT_EVENT_CREATED));
+        } catch (Exception e) {
+            log.error("Failed to create Move Team to New Unit audit event for team UUID {} for reason {}", teamUUID, e, value(EVENT, AUDIT_FAILED));
+        }
+    }
+
+    public void createTopicAudit(Topic topic) {
+        String auditPayload = Json.createObjectBuilder()
```

Comment:
I like this approach to building the audit payload, we should do this for the other services